Fixes #456854.
qiv: FTBFS: error: gdk/gdkx.h: No such file or directory


--- ../orig/qiv-2.1~pre12/Makefile	2007-04-25 02:52:39.000000000 +0200
+++ Makefile	2007-12-20 09:49:03.000000000 +0100
@@ -66,6 +66,7 @@
 #	    -fthread-jumps #-march=pentium #-DSTAT_MACROS_BROKEN
 
 INCLUDES  = `imlib-config --cflags-gdk`
+INCLUDES += `gtk-config --cflags`
 LIBS      = `imlib-config --libs-gdk`
 # [as] thinks that this is not portable enough
 # [lc] I use a virtual screen of 1600x1200, and the resolution is 1024x768,
